### PR TITLE
Add event results API

### PR DESCRIPTION
**Pull request by wiedmann**
This adds the ability to pull the Zwift results by event subgroup ID.

### CODE REVIEW

File: src/World.js [hunk 1]
```diff
@@ -36,6 +38,26 @@ class World {
                 })
             })
     }
+
+    segmentResults(eventSubgroupId) {
+        return this.request.protobuf(`/api/segment-results?world_id=0&segment_id=1&event_subgroup_id=${eventSubgroupId}&full=true&player_id=0`)
```
Comment (Ogadai) [inline src/World.js:43]: I guess `world_id`, `segment_id` and `player_id` are just ignored. Does it still work if those parameters are left out?

`world_id` is actually passed into the constructor, so possibly should be `world_id=${this.worldId}` to be consistent (even though it's always 1 and probably ignored).

**Review by Ogadai — COMMENTED** [review]
How can I get hold of a valid `eventSubgroupId` so I can try this out?

**Comment by Ogadai** [comment]
Also, do you want to add to the ReadME to show usage?

File: src/World.js (hunk 1 above)
Comment (wiedmann) [reply src/World.js:43]: I haven't tried leaving them out, but this is the way the Zwift client sends the request. I think segment id is probably required to be 1. This world ID doesn't appear to be the same as passed to the constructor. 0 may be a wild-card or something. I'm not sure about the player id.

Eventually I think I would want to add the ability to query segments as well as events, at which point the parameters may be needed.

**Comment by wiedmann** [comment]
Yes, I can add to the ReadMe as well.

**Review by Ogadai — APPROVED** [review]

**Pull request merged**